back/app/models/user.rb

```diff
@@ -363,6 +362,10 @@ def authenticate(unencrypted_password)
     end
   end
 
+  def no_password?
```

Comment:
Have you implemented that in your branch? Can we not still allow passwordless auth if someone has registered with google or facebook? Not sure we came to any conclusion on that.

Reply:
Yes, I currently implemented it like this. I'm not sure if I understand the second question, are you saying we should treat users who signed up with Google/Facebook as light users? This would mean than many of the existing users would suddenly get restrictions on where they can participate.

Reply:
But it is going depend less on how they've authenticated and more upon which data we need for each action, but I think probably it's right to put the restriction in place now but assess in the coming iterations

Reply:
I've added the restriction, but kept the naming as no_password? as it felt nicer - especially as I've added a no_name? method in my other branch and nameless didn't sound right.